packages/api/server/ws.mts

```diff
@@ -682,6 +686,47 @@ async function tsconfigUpdate(payload: TsConfigUpdatePayloadType) {
   });
 }
 
+async function tsserverQuickInfo(payload: TsServerQuickInfoRequestPayloadType) {
+  const session = await findSession(payload.sessionId);
+
+  if (!session) {
+    throw new Error(`No session exists for session '${payload.sessionId}'`);
+  }
+
+  if (session.language !== 'typescript') {
+    throw new Error(`tsserver can only be used with TypeScript Srcbooks.`);
+  }
+
+  const tsserver = tsservers.has(session.id) ? tsservers.get(session.id) : createTsServer(session);
```

Comment:
Scratch that, I think I'm wrong here. Nothing would prevent users from hovering over code and sending this to the backend when a new srcbook is either created or imported before deps are installed (because tsserver wouldn't be available yet).

If deps are not yet installed then we don't want this to fail / break. We probably need to ignore failures here?